### ISSUE TITLE
Tab count out of sync for removed tabs

### ISSUE THREAD

**Issue by jonathanKingston** [jonathanKingston/sea-containers#11]

**Comment by Smile4ever** [comment]
Did you add an event handler for https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/onRemoved?

**Comment by jonathanKingston** [comment]
Yeah it deletes the tab fine, I just didn't recount the tabs. It's an easy fix however I wanted to rethink how the parent UI is rendered.

The first release of this actually worked completely fine, it just rerendered the DOM like alllll the time however I want to make #22 better not worse.

**Comment by bhearsum** [comment]
This is what causes the count to be wrong until closing and re-opening the sidebar?

**Comment by mloki** [comment]
Hi Jonathan, 

First thanks for the time you're putting on the addon. It's really good but there's room for 
I cloned the repo and I'm trying to find the code to fix the tab count but I'm not very familiar with the WebExtensions API. Could you give me some pointers on where the code is. 
Also I believe this is related to #6 

Cheers,
mloki

Edit: Also possible related to #45
